servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.h

```diff
@@ -155,8 +155,7 @@ class RenderForwardClustered : public RendererSceneRenderRD {
 
 	uint64_t lightmap_texture_array_version = 0xFFFFFFFF;
 
-	bool base_uniform_set_updated = false;
-	void _update_render_base_uniform_set();
+	void _update_render_base_uniform_set(const RendererRD::MaterialStorage::Samplers &samplers);
```

Comment:
```suggestion
	void _update_render_base_uniform_set(const RendererRD::MaterialStorage::Samplers &p_samplers);
```
Prefix arguments with `p_`